mffpy/reader.py

```diff
@@ -150,7 +154,8 @@ def _blobs(self) -> Dict[str, bin_files.BinFile]:
         for si in self.directory.signals_with_info():
             with self.directory.filepointer(si.info) as fp:
                 info = XML.from_file(fp)
-            bf = bin_files.BinFile(si.signal, info)
+            bf = bin_files.BinFile(si.signal, info,
```

Comment:
So, just to make sure I'm on the correct page - since the method documentation in `bin_files.py` leavea a bit to be desired.

When initialized an instance of the bin file class, if the type of the signal isn't specified - it defaults to EEG. Therefore, but reading the channel_type field from each bin file as its being read, we can assure all bin file types are being touched upon.

Do we know whether non-EEG bin files are being correctly imported?

Reply:
Yes, that's right. Specifying channel_type separates out the signals by the channel_type label, so unless there are somehow multiple 'EEG' type bin files, we should be reading all bin files. There's also an option to specify a specific channel type in `get_physical_samples()` if you want to just grab the 'EEG' channels or something else.

As far as I can tell, non-EEG files are being read correctly. The number of channels and number of signals checks out (sampling rate for all types of channels is the same which makes it easy).